database/clickhouse/examples/migrations/003_create_database.up.sql

```diff
@@ -0,0 +1,81 @@
+CREATE DATABASE IF NOT EXISTS analytics;
+
+CREATE TABLE IF NOT EXISTS analytics.driver_ratings(
```

Comment:
Any reason you chose these tables and various engines? I prefer the DDLs to be as simple as possible to avoid any future integration test failures.

Reply:
That's good, but as a user of the migration package, I've run into trouble writing more complex migrations than the ones in the example, and I believe having this example will help people design their own migrations. In addition to the fact that this example solves the main problem with describing the creation of a database within a migration, it is also a real example that I used for my project to create a connection between a Kafka partition and a Clickhouse database. The example only seems complicated, in fact, it is a simple creation of different tables without replication to the cluster, and the use of different engines should not affect the addition of integration tests, since people actively use them.